### PR TITLE
Add more padding on sides of home page and in name dropdown

### PR DESCRIPTION
**Pull request by jhanstra**
Goes with https://github.com/UW-Madison-DoIT/uw-frame

### CODE REVIEW

**Comment by vertein** [comment]
:+1:

**Comment by keirserrie** [comment]
:+1:  Jared--while you're in here could you make widget bg white instead of gray? Will give much nicer contrast there.

File: angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/search-results.html [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
 			</li>
 			<li>
-				<a>Directory (# results)</a>
+				<a>Directory (#)</a>
 			</li>
 			<li>
-				<a>Wisc.edu (# results)</a>
-			</li> -->
+				<a>Wisc.edu (#)</a>
+			</li>
```
Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/search-results.html]: Did you mean to do this as part of this?

Comment (jhanstra) [inline angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/search-results.html]: Nope, good catch. I'll uncomment out for the time being.

**Comment by jhanstra** [comment]
@keirserrie sure, the code is in the other pull request in uw-frame, but here's a screenshot of it now.
image

File: angularjs-portal-home/src/main/webapp/css/home.less [hunk 2]
```diff
@@ -290,127 +290,11 @@ div.table-cell {
...
-  border-radius:4px;
-}
-.simple-content-container {
-  height:280px;
-  overflow:hidden;
-}
-.widget-simple-html {
-  height:210px;
-  overflow:scroll;
-}
```
Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/css/home.less]: Pumped about deleting all this css

**Comment by timlevett** [comment]
:+1:

**Pull request merged**